src/api/exceptions.cc

```diff
@@ -231,8 +236,9 @@ Local<Value> WinapiErrnoException(Isolate* isolate,
         .FromJust();
   }
 
-  if (must_free)
-    LocalFree((HLOCAL)msg);
+  if (must_free) {
+    LocalFree(HLOCAL(msg));
```

Comment:
Count you try
```suggestion
    LocalFree(reinterpret_cast<const HLOCAL>(msg));
```

Reply:
Done :)

Reply:
I think we should leave this with `// NOLINT(google-readability-casting)` because there is a bug in the API and what we really need here is:
```c++
    LocalFree(const_cast(char*>(msg));
```
but that's just as bad for `tidy`.